CHANGELOG.md

```diff
@@ -8,6 +8,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 
```

Comment:
@ThilinaManamgoda as per evaluation, it was noticed that this product profile is called, `micro-integrator monitoring dashboard`. Shall we name the Dockefile source directory with an appropriate term like `mi-monitoring-dashboard` to map with the product profile rather than using `analytics` although the meanings are similar?

Reply:
The product pack is named as **wso2ad-1.0.0**. I think the "micro-integrator monitoring dashboard" is a different usecase[1]. 

[1]https://ei.docs.wso2.com/en/7.1.0/micro-integrator/administer-and-observe/working-with-monitoring-dashboard/